debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSchema.java

```diff
@@ -35,7 +35,7 @@
 /**
  * Component that records the schema information for the {@link PostgresConnector}. The schema information contains
  * the {@link Tables table definitions} and the Kafka Connect {@link #schemaFor(TableId) Schema}s for each table, where the
- * {@link Schema} excludes any columns that have been {@link PostgresConnectorConfig#COLUMN_BLACKLIST specified} in the
+ * {@link Schema} excludes any columns that have been {@link PostgresConnectorConfig#COLUMN_EXCLUDE_LIST} specified in the
```

Comment:
The orginal closing brace position made more sense actually. This will be rendered as a link labeled "specified".

Reply:
ok. will do

Reply:
Done